### ISSUE TITLE
Is it possible to run lss within a web page?

### ISSUE THREAD

**Issue by andyburnett** [ympbyc/LittleSmallscript#35]
I have just stumbled across this project, and I think it is great. What I would really like to do is to have a version of lss as a chrome extension which I can inject into any web page. This would then allow me to scrape the page, or do whatever I want, but in Smalltalk.  Is this possible, or does the design fundamentally require Node running?

**Comment by kdriedger** [comment]
@andyburnett you may consider looking at amber smalltalk ( http://amber-lang.net/ ).  I've seen them embed it into a webpage.   This may require some Chrome Developer Tools expertise.

**Comment by sebastianconcept** [comment]
@andyburnett have you seen https://bertfreudenberg.github.io/SqueakJS/ ?

**Comment by andyburnett** [comment]
Hi Sebastian,

Thanks for the suggestion. I looked at Bert's work a while ago. It is
amazingly impressive!

I didn't follow up on it, last year, because it still seemed too
experimental. However, I have just looked at it again, and it has made big
progress. I can see how I might be able to use it the way I want.
Basically, I would like inject it into any webpage, so that I could do data
manipulation via smalltalk.

Time for some experiments.

Cheers
Andy
